ui/app/src/components/DateTimePicker.tsx

```diff
@@ -149,10 +149,10 @@ const DateTimePicker = withStyles(dateTimePickerStyles)((props: DateTimePickerPr
     }
 
     setSelectedDateTime(updatedDateTime);
-    onChange?.(updatedDateTime);
+    onChange && onChange(updatedDateTime);
```

Comment:
What’s the falsy value that comes on onChange and where does it come from? Why is it something other than null/undefined?

Reply:
@rart so, should onChange be required so I can assume it will always be different than null/undefined?

Reply:
I reverted the optional chaining by mistake, changed back.